### PR TITLE
Status bar

### PR DESCRIPTION
**Pull request by p4cx**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
New status bar in EMBA to improve nerd factor and usability (yes in that order 😆) 
Screenshot 2022-07-20 at 23 29 36

* **Other information**:
Usage and other informations will be added to the wiki.

### CODE REVIEW

File: modules/S115_usermode_emulator.sh [hunk 1]
```diff
@@ -259,7 +259,7 @@ prepare_emulator() {
     chmod +x "$R_PATH"/fixImage_user_mode_emulation.sh
     cp "$(which busybox)" "$R_PATH"/
     chmod +x "$R_PATH"/busybox
-    chroot "$R_PATH" /busybox ash /fixImage_user_mode_emulation.sh | tee -a "$LOG_PATH_MODULE"/chroot_fixes.txt
```
Comment (m-1-k-3) [inline modules/S115_usermode_emulator.sh:262]: do we really need to change this?

File: helpers/helpers_emba_status_bar.sh [hunk 2]
```diff
@@ -0,0 +1,412 @@
...
+export PID_STATUS=""
+export PID_MODULES=""
+export PID_STATUS_2=""
+# Path to status tmp file
+# each line is dedicated to a specific function
+# 1: Count of boxes visible
+# 2: CPU load string (needs to be cached, because it takes about a second to get the information)
+# 3: Start time of status bar - not exactly the same as the EMBA timer, but near enough to get an idea
+# 4: Count modules
+export STATUS_TMP_PATH=""
```
Comment (m-1-k-3) [inline helpers/helpers_emba_status_bar.sh:29]: why init all of this here and not in the initial_status_bar?

File: helpers/helpers_emba_status_bar.sh [hunk 3]
```diff
@@ -0,0 +1,412 @@
...
+
+# helper for box drawing
+repeat_char(){
+  local REP_CHAR="$1"
```
Comment (m-1-k-3) [inline helpers/helpers_emba_status_bar.sh:33]: Shouldn't we use "${1:-}" in function parameters?

File: helpers/helpers_emba_status_bar.sh [hunk 4]
```diff
@@ -0,0 +1,412 @@
...
+  local REP_COUNT="$2"
+  local RET=""
+  local A=0
+	for ((A=1; A<=REP_COUNT; A++)) ; do RET+="$REP_CHAR"; done
```
Comment (m-1-k-3) [inline helpers/helpers_emba_status_bar.sh:37]: indent?

**Review by m-1-k-3 — CHANGES_REQUESTED** [review]
master piece of bash ... love it
Please check my comments

File: modules/S115_usermode_emulator.sh (hunk 1 above)
Comment (p4cx) [reply modules/S115_usermode_emulator.sh]: no, I reversed it

File: helpers/helpers_emba_status_bar.sh (hunk 2 above)
Comment (p4cx) [reply helpers/helpers_emba_status_bar.sh]: moved it

File: helpers/helpers_emba_status_bar.sh (hunk 3 above)
Comment (p4cx) [reply helpers/helpers_emba_status_bar.sh]: yes, changed it

File: helpers/helpers_emba_status_bar.sh (hunk 4 above)
Comment (p4cx) [reply helpers/helpers_emba_status_bar.sh]: fixed it

**Review by m-1-k-3 — APPROVED** [review]

**Pull request merged**